nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -530,6 +530,17 @@
       The <literal>nodejs-11_x</literal> package has been removed as it's EOLed by upstream.
      </para>
    </listitem>
+   <listitem>
+     <para>
+       Because of the systemd upgrade,
+       <program>systemd-timesyncd</program> will no longer work correctly if
```

Comment:
can we delete `correctly` from here and the next line?

Reply:
Yes, will do the first, but I'm against removing the second — it won't work if it's set to "19.09" for instance.

Reply:
I've reworded this a bit more.